### PR TITLE
dp/fix reorg default

### PR DESCRIPTION
**Pull request by DenhamPreen**

### CODE REVIEW

File: static/schemas/config.evm.json [hunk 1]
```diff
@@ -69,18 +57,12 @@
       ]
     },
     "rollback_on_reorg": {
-      "description": "A flag to indicate if the indexer should rollback to the last known valid block on a reorg (default: false)",
-      "type": [
-        "boolean",
-        "null"
-      ]
+      "description": "A flag to indicate if the indexer should rollback to the last known valid block on a reorg (default: true)",
+      "type": ["boolean", "null"]
```
Comment (DenhamPreen) [inline static/schemas/config.evm.json:61]: This is the only change the rest is formatting

**Comment by DenhamPreen** [comment]
@JonoPrest this looks like an auto generated file however I couldn't see anything in the readme about where or how its generated. 

I suspect this PR is not the complete solution.

cc: @JasoonS if you know more

**Comment by DenhamPreen** [comment]
Thanks Dmitry, I will do. Do you know the command / could someone who does add the command into the readme in this branch to autogenerate this file?

**Comment by JonoPrest** [comment]
> @JonoPrest this looks like an auto generated file however I couldn't see anything in the readme about where or how its generated.
> 
> I suspect this PR is not the complete solution.
> 
> cc: @JasoonS if you know more

Hey Den, looks like Dmitry got back to you. It's an autogenerated schema. And the script you will find in package.json

It runs as a prebuild so should do automatically just like the other scripts (although I actually don't love that)

**Pull request closed without merging**